resources/js/Components/Table/CharacterTable/CharacterTable.tsx

```diff
@@ -96,7 +96,7 @@ const CharacterTable = ({
 
     const handleSelectAllClick = (event: any) => {
         if (event.target.checked) {
-            const newSelecteds = rows.map((n) => n.cname)
+            const newSelecteds = rows.map((n) => n.name)
```

Comment:
i don't see why map is needed, if select all is just selecting all rows

Reply:
It seems that rows are set based on the 'selected' array of strings. 'newSelected' contains a list of names of all the characters.  Do you know of another way of strictly extracting the 'names' from the 'rows' array?
![image](https://user-images.githubusercontent.com/35549905/139557683-0f4d4378-70f5-44d5-890e-1112ddbe6faf.png)